src/Build.OM.UnitTests/AssemblyInfo.cs

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+global using NativeMethodsShared = Microsoft.Build.Framework.NativeMethods;
```

Comment:
I appreciate trying not to make the diff bigger than necessary, but this is also slightly misleading, as it's in Framework, not Shared. That doesn't bother me too much since Framework is "shared" just not Shared if that makes sense, but it's a little confusing.

Reply:
Yeah, the implications of changing this everywhere made me pretty sad so this seemed like a good compromise.